.travis.yml

```diff
@@ -12,7 +12,8 @@ after_success:
-  - secure: pnBYrvZ7pP3DcdOL6CTNiia9mwHrEXLuIy/oiXp5eHz0gr3PThMtlzE9XqKGYZJjR5C1RkqnyR9VA8IzU39yjZYSIbWZR2ArTUr2bzREv9TmRpXxoTHGZsfS5KX6mENHgdhkV7DLsNW7MQXAuQd61NetarABjK2eMVqkd6R76JFLNXcN61FZGaQ0PjxOteh7V6ZUY0NAYq4Pc9fMA3urxgDytX1f7RBOtvNDga38xcUzMi7eMcEL/t6zWwlqbcqdvEU6nBXIF2rFUw2K7RFb7zpXCcWdzDmfW4JJnGPKjb/9X5sUe4KA64khYx7+9FqpcTna1Pse6D47iMDuz7RnhaZyXTNeHSZq+WrxEct8VFUHAYgseldT9EMZjCRRj9yAAJTRPgj+gYNQSGciFFWyYXy2s1egSFKNNHaNlxR38wb6azDBs5YHZlp6z823NBnPa9WlSlBEWw4cuyuX/7A565ayEU2hZFlc1aRCoxsSKMbOE8ypv/wz0L2oCsjlaq9wxBC2oh8rF6PQyC9UgphX48uXYr/Ff7+i9hSZVhR0YgoVayyQHWMxlNzehI+xihNlhjiyRR2oHwRFuY898eGupwXYDVIVI52O20AgQ/zgNZraNzYEUwTNBlvgXSW9TGZnxVWWc7zmIImKioVCuOxTwwLW9ktpTr5kFha9/gVKAT0=
-  - secure: wshr//K3w0IUqlgOT2UP/U3IMmy4gZ6JL3vRSt7rfTT1oBIMFb1KJWFk+gqEB3otAv2WtbHGc/8fFK/d4cV/ZyuYPrhcGmdbjDxU/juzYa5dSm9qF2zteasspWl9gTyPyaBWx7TGOB7uyDjK+SmS/7MM/l/hjSl5W6GtoE1wVph8pLgB9ubIDDAEv1q8ttmYsBckoFHcufYnFO749K6F89mmShFiapniRpqYG9eMSBwx3cH+PdtWANNENrIfWjmo31Kg3JfOOifTzuSHOGh/zmcZWaXhFAxyguxJc5ARVyfXV127wa8ZHX6EDyZaS0I72lNYr7oagBY8oO4h73MpHODIo1rde2iFIwmvBTVWAk8oS0CSx2+tR9ZGsGF2o0moOCS37nIs7OrwqzUbuujPBaMgL6IpOLUoVWXFAUYjdrk4s3FgOSa5MGqjOCzekumNTeiYb2BdXvh3Wpc7biFZnpsNZ0EXAhUKBApThvNJknSHZfR8JxAMUCXe4MKwp3VdpKWoHEzX6kMyw4ddzEWxHHp3TfqoFBfCin3+V8erfJWHZZ5meYNQfqPHHWvRVSg1Rz6E6yK9KTfVvyFqKL285n7fzz3hz7SWDL9Sx8/nkHS+Dfh2E9b32N4165+bwHoik32+/uIHDLl66vvLOqE8vVbgSHWIFqg7fH6JGadMjGM=
-  - secure: VhCo8v1kn2Qtfqh6hzUHgd5GJ2U6YWGSpUOfPy513zkS6dMl0Hxp1Sb85nDfBwD72IUzMJst87oK+9yDX60GdJqAkh3L72mI9nUUEM9W4ag3iLpXX0ZLPct4qmUWujLtoVrnI9HP7FgY3C20Vs4/aFoBMcPG2vxHP3fty8SRWNpk944tOf5c1G+TzXutxrjLBr1FssW2FDoaDF/f3vc/IjJ06I5C0a9ARNiFTPeuZxo8iX66Y3iZOITejvPk0zwtGnEpZeDQ23kz1oQg86lYfMyCBqL/hnKaG7uCd1pt8HxnWpJUX6jpE2KewrmP5xv/v6RjC5qIsxD84qz3jlub6S8SXL3aaYPxDYoV1EpPDwmjmVh2mHB3TFOrwjaMtZVJUhzC8zE3Xlr/CD38GwPuuqzGDoluLvBnSXOOJW9+RDMp7vjy+dh9WahtX6k4sT3BbSRl1mHe23UfQG5j+Qu7LifDzQINySmJKjjZ9PYEs4ZVl4xFSnXDQuVJAG6CxBy3a3KgJrdGUK3TxAE9lJrDT/iiapiq/au4JzmXXvPuCsIxVNLbEcXkXcxuO3MkqIv1+nLOE8dYg3vOCG+2hQrXzb7qCw74so3jdiafqCRooDTVD4zlB1b8MTj+2t+FoI1mpueNKvEFtBYv1eMq4pJ1vwJNJh4EVAHPixkCZi1/gD8=
+  - secure: Vbf88W9HlfliMD+BGsCm3bW31kmrG6i2NHxTCdhYuJhKDt0lOCUMiNG7RSNT9xeaG4xBwlr/G8bn3gzhLjhdPlE1ETgrjdhemj9G8X18LJkscKQ0rw/mi0wWV28p8DOylVHenQwlANOQ+BZxBQbLgovmgdpl8bI0ADfaaf58QlQSSCCc4Hb1gv+XJSwu7WKWqLAwD6sgC8gTobICBzQm+Cd8tbK/eBggMA3Vh4ZTmfvmD/aL++Nl8hEWwvuP9RHB0jItSt+1PLmBQ/HPwxj/2sM0OwTJwXIGyQ4wzNUGt8/whCvWugmaNsC/FMMGQB4tbfYiKNigHdKzPzovA/LaXijoK2OYJMb/uaOfrue8C1J5Lu7FExkuIQSvk4yic8wAV1GQ4J9+xsb0XiTh4Ecd0VoGxyIGGCGLprp/v1mc++Xo7xX+5zLlhlWjCUAuB9xt2ZG5lCqVbkXcxvLwO2pfwm26nB5WBdIFflUy05fWhBGqqwvZeUO5LPyLC60h8/Gn0/pzea7xSFQUHeLt8EwZ26K1Tf3j+DUTJIeIMgLScKPgj6Fz2byD5KtDz8LmMPRG7oD1TLW/mmMGYa5eeu+oje+oFnRxJyuLy3X9R61lTPIzcjJjsK9R2SoblAt1+K1nPD9u7fdBh64T2HX6objbj6waNPrrLYWR1JFrJ6Af/Ro=
```

Comment:
Why did these need to change?

Reply:
The API key was not in there. This had not been lit up because it was running `mvn test` rather than `mvn verify`. I just recreated them all to be safe.

Reply:
> I just recreated them all to be safe.

You probably didn't need to do that, but :+1:.  Just wanted to make sure it was deliberate.